src/main/java/DataManipulation.java

```diff
@@ -0,0 +1,81 @@
+import static java.util.Arrays.asList;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.FieldVector;
+import org.apache.arrow.vector.VarCharVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+
+public class DataManipulation {
+  public static final RootAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+
+  public List readCSVFile(String path) throws FileNotFoundException {
+    String line;
+    final VarCharVector nameVector = new VarCharVector("name", allocator);
+    final VarCharVector teamVector = new VarCharVector("team", allocator);
+    final VarCharVector positionVector = new VarCharVector("position",allocator);
+    final VarCharVector heightVector = new VarCharVector("height", allocator);
+    final VarCharVector weightVector = new VarCharVector("weight", allocator);
+    final VarCharVector  ageVector = new VarCharVector("age", allocator);
+    try (BufferedReader br = new BufferedReader(new FileReader(path))) {
+      int i = 0;
+      while ((line = br.readLine()) != null) {
+        String [] values = line.split(",");
+        nameVector.setSafe(i, values[0].getBytes());
+        teamVector.setSafe(i, values[1].getBytes());
+        positionVector.setSafe(i, values[2].getBytes());
+        heightVector.setSafe(i, values[3].getBytes());
+        weightVector.setSafe(i, values[4].getBytes());
+        ageVector.setSafe(i, values[5].getBytes());
+        ++i;
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    List<FieldVector> list = asList(nameVector, teamVector,
+        positionVector, heightVector, weightVector, ageVector);
+    return list;
+  }
+
+  public VectorSchemaRoot vectorToAVectorSchemaRoot(List list){
+    final VectorSchemaRoot vectorSchemaRoot = new VectorSchemaRoot(list);
+    vectorSchemaRoot.setRowCount(1035);
```

Comment:
Should not rely on hardcoded number of rows, if we change the CSV the program will not work correctly